### DISCUSSION TITLE
not able to login- user ip address front end 52.91.97.200 | user 18.232.250.137

### DISCUSSION THREAD

**Discussion by Nazrina412** [learndevopsonline/learndevopsonline#1930] [Q&A]
![Screenshot 2024-07-14 135006](https://github.com/user-attachments/assets/10e8b898-8302-4b1c-ad87-e4c62270e157)

**Comment by chaithanya1812** [comment]
Hi @Nazrina412 , you user service is working fine. I just restarted the nginx once..
![image](https://github.com/user-attachments/assets/0de7540a-8703-4167-9c9b-6ce15783392e)

**Comment by Nazrina412** [comment]
thank you
